src/config/setup-container.mjs

```diff
@@ -54,8 +60,12 @@ export const injectables = {
   // feature-specific lib (homemade utilities)
```

Comment:
Should these always go here? Or should the files that define them also inject them. I can imagine this list of imports and registrations gets pretty big over time. I guess I'm looking for something like spring where you can annotate the method where it's defined, and it gets registered automatically. Any pattern for that?

Reply:
awilix does have auto-discovery tooling; i haven't played with it any yet.